### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by iprankurpandey**
Things to watch : 
Sidebar 
A product cart with two buttons

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
code looks good, just some changes which I think can be done mentioned below, feel free to merge

File: agri-ecom/src/components/Produclisting/ItemCard.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <img className="vcard-image-inside" src="/" alt="scorpio" />
+        <div className="card-content">
+          <div className="head-with-icon">
+            <h3>Scorpio</h3>
+            <span className="material-icons mright" title="verfied">
+              check_circle
+            </span>
+          </div>
+          <div className="text">The Car</div>
+          <span className="text-light">By : Astro</span>
```
Comment (nikhiljugale007) [inline agri-ecom/src/components/Produclisting/ItemCard.js:16]: try not use span as it is not good for accessibility.

**Review by nikhiljugale007 — COMMENTED** [review]
overall code is good. added a suggestion. 
one more suggestion try to remove commented code.

File: agri-ecom/src/components/SidebarFilter/SideBarFilter.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          <hr />
+          <div className="price-range">
+            {" "}
+            Price Range
+            <input type="range" min={25} max={100} value={75} />{" "}
+          </div>
+          <hr />
+          <h3 class="catagories-sort-by">Catagories sort by : </h3>
+          <input type="checkbox" name="" id="" />
+          Machine <br />
```
[the remark this answers is not in the document]
Comment (iprankurpandey) [reply agri-ecom/src/components/SidebarFilter/SideBarFilter.js:28]: Sure will fix this now

File: agri-ecom/src/components/Produclisting/ItemCard.js (hunk 1 above)
Comment (iprankurpandey) [reply agri-ecom/src/components/Produclisting/ItemCard.js:16]: Thanks for the review will fix this now

**Pull request merged**